pkg/asset/agent/installconfig.go

```diff
@@ -99,9 +100,16 @@ func (a *OptionalInstallConfig) validateSupportedPlatforms(installConfig *types.
 	if installConfig.Platform.Name() != none.Name && installConfig.ControlPlane.Architecture == types.ArchitecturePPC64LE {
 		allErrs = append(allErrs, field.Invalid(fieldPath, installConfig.Platform.Name(), fmt.Sprintf("CPU architecture \"%s\" only supports platform \"%s\".", types.ArchitecturePPC64LE, none.Name)))
 	}
-	fieldPath = field.NewPath("Platform", "External", "PlatformName")
-	if installConfig.Platform.Name() == external.Name && installConfig.Platform.External.PlatformName != "oci" {
-		allErrs = append(allErrs, field.NotSupported(fieldPath, installConfig.Platform.External.PlatformName, []string{"oci"}))
+	if installConfig.Platform.Name() == external.Name {
+		if installConfig.Platform.External.PlatformName != string(models.PlatformTypeOci) {
+			fieldPath = field.NewPath("Platform", "External", "PlatformName")
+			allErrs = append(allErrs, field.NotSupported(fieldPath, installConfig.Platform.External.PlatformName, []string{string(models.PlatformTypeOci)}))
+		}
```

Comment:
@zaneb should this be removed? In 4.15, the platform name can be other than OCI but we yet do not know what those other values are. So I believe we may keep it as is for now and need to revisit it later when we know what are the other valid values for the platform name in case of external.

Reply:
We'll need to remove it to complete [AGENT-704](https://issues.redhat.com/browse/AGENT-704), but I would agree with leaving it for now. [MGMT-15654](https://issues.redhat.com/browse/MGMT-15654) is a pre-requisite, so it's probably best to wait until we know for sure when that will land.